src/cdk-experimental/menu/menu-item-trigger.ts

```diff
@@ -27,11 +28,32 @@ import {
   FlexibleConnectedPositionStrategy,
 } from '@angular/cdk/overlay';
 import {SPACE, ENTER, RIGHT_ARROW, LEFT_ARROW, DOWN_ARROW, UP_ARROW} from '@angular/cdk/keycodes';
+import {Subject, merge} from 'rxjs';
+import {takeUntil} from 'rxjs/operators';
 import {CdkMenuPanel} from './menu-panel';
 import {Menu, CDK_MENU} from './menu-interface';
-import {FocusNext} from './menu-stack';
+import {FocusNext, MenuStack} from './menu-stack';
 import {throwExistingMenuStackError} from './menu-errors';
 
+/**
+ * Whether the target element is a menu item to be ignored by the overlay background click handler.
+ */
+export function isClickInsideMenuOverlay(target: Element): boolean {
+  while (target?.parentElement) {
+    const isOpenTrigger =
+      coerceBooleanProperty(target.getAttribute('aria-expanded')) &&
```

Comment:
```suggestion
    const isTriggerOpen = target.getAttribute('aria-expanded') === 'true';
```
I would use a direct comparison to `'true'` here since `coerceBooleanProperty` will treat the empty string as `true` since it's intended to match the behavior of HTML boolean attributes